plinth/modules/config/config.py

```diff
@@ -51,6 +53,16 @@ def get_domainname():
     return '.'.join(fqdn.split('.')[1:])
 
 
+def get_language(request):
+    """Return the current language setting"""
+    # TODO: Store the language per user in kvstore,
+    # taking care of setting language on login, and adapting kvstore when
+    # renaming/deleting users
+
+    # The session contains more accurate information than request.LANGUAGE_CODE
+    return request.session[translation.LANGUAGE_SESSION_KEY]
```

Comment:
I get a KeyError here on the first run, probably because the language has not been set yet. It seems like we can just return None in case of the KeyError though.